tests/posix/fs/prj.conf

```diff
@@ -5,4 +5,5 @@ CONFIG_POSIX_API=y
 CONFIG_POSIX_FILE_SYSTEM=y
 CONFIG_ZTEST=y
 CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_ZTEST_STACK_SIZE=2048
```

Comment:
You say 1-4k but you are in fact increasing to 2k

Reply:
Fixed (the commit message)